frontend/src/components/pages/CreateReview/CreateReviewPage.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button } from "@chakra-ui/react";
+import React, { useState } from "react";
+
+import { Book } from "../../../types/BookTypes";
+import { Option } from "../../../types/TagTypes";
```

Comment:
hmmm, maybe build aliases to avoid these intense `../../../`?

Reply:
Ofc that's outside of the scope of this PR, but just a suggestion to work on down the road.